pandas/core/computation/pytables.py

```diff
@@ -216,17 +215,16 @@ def stringify(value):
             if isinstance(v, (int, float)):
                 v = stringify(v)
             v = ensure_decoded(v)
-            v = Timestamp(v)
-            if v is not NaT:
```

Comment:
Is this thanks to #48819, which introduced `NaT.as_unit`?

Reply:
correct